src/Illuminate/Foundation/Auth/AuthenticatesAndRegistersUsers.php

```diff
@@ -1,115 +1,60 @@
 <?php namespace Illuminate\Foundation\Auth;
 
 use Auth;
```

Comment:
If you refer to the line above, I didn't touch it cause its not mine, its in the existing code.